### PR TITLE
Diana/spaceinvaders

### PR DESCRIPTION
**Pull request by Dru27**
UI is working with aliens, ship, bullet and blocks.

### CODE REVIEW

File: git-playground/Agustin/userinfo [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-Licenciatura en informatica / Analista Programador Universitario
-La Plata
-user code: agustin-1996
-month code: agustin-04
+Agustin Mediotti
+06/04/1996
+Licenciatura en informatica / Analista Programador Universitario
+La Plata
+user code: agustin-1996
+month code: agustin-04
```
Comment (adhes14) [inline git-playground/Agustin/userinfo:6]: Weird, there seems to be no change

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Block.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import static org.fundacionjala.at15.spaceinvaders.Constants.Block.*;
+import java.awt.Graphics;
+import java.awt.Color;
+
+public class Block {
+    private int lifeAsteroid;
+
+    public Block(int life) {
+        this.lifeAsteroid = life;
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Block.java]: should be asteroidLife

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java [hunk 3]
```diff
@@ -7,25 +7,31 @@
 import java.util.ArrayList;
 import java.util.List;
 
+
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java]: there is an unnecessary line break

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java [hunk 4]
```diff
@@ -41,8 +47,8 @@ public void createAliens(int rows, int columns) {
         for (int yIndex = 0; yIndex < rows; yIndex++) {
             for (int xIndex = 0; xIndex < columns; xIndex++) {
                 Alien alien = new Alien(
-                    ALIEN_INIT_X + ALIEN_SEPARATION * xIndex,
-                    ALIEN_INIT_Y + ALIEN_SEPARATION * yIndex, ALIEN_WIDTH, ALIEN_HEIGHT);
+                        ALIEN_INIT_X + ALIEN_SEPARATION * xIndex,
+                        ALIEN_INIT_Y + ALIEN_SEPARATION * yIndex, ALIEN_WIDTH, ALIEN_HEIGHT);
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java:49]: there are unnecessary spaces

**Review by adhes14 — CHANGES_REQUESTED** [review]
please check my comments

File: git-playground/Agustin/userinfo (hunk 1 above)
Comment (Dru27) [reply git-playground/Agustin/userinfo:6]: I agree, it seems strange to me that there has been any change.

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Block.java (hunk 2 above)
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/spaceinvaders/Block.java]: Thank you! It's a grammar error, I'll correct it.

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java (hunk 4 above)
Comment (Dru27) [reply src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java:49]: They are not spaces, the code editor saves like this when the line is very long.

**Review by adhes14 — APPROVED** [review]

**Pull request merged**